native/src/main/java/io/ballerina/stdlib/ftp/transport/server/RemoteFileSystemConsumer.java

```diff
@@ -88,20 +88,20 @@ public RemoteFileSystemConsumer(String id, Map<String, String> fileProperties, R
             throw new RemoteFileSystemConnectorException(
                     "[" + serviceName + "] Unable to initialize " + "the connection with server.", e);
         }
-        if (fileProperties.get(Constants.FILE_NAME_PATTERN) != null) {
-            fileNamePattern = fileProperties.get(Constants.FILE_NAME_PATTERN);
+        if (fileProperties.get(FtpConstants.FILE_NAME_PATTERN) != null) {
+            fileNamePattern = fileProperties.get(FtpConstants.FILE_NAME_PATTERN);
         }
     }
 
     private void validateParam(Map<String, String> fileProperties) throws RemoteFileSystemConnectorException {
-        if (fileProperties.get(Constants.URI) == null) {
+        if (fileProperties.get(FtpConstants.URI) == null) {
             final RemoteFileSystemConnectorException e = new RemoteFileSystemConnectorException(
-                    Constants.URI + " is a mandatory parameter for FTP transport.");
+                    FtpConstants.URI + " is a mandatory parameter for FTP transport.");
             remoteFileSystemListener.onError(e);
             throw e;
-        } else if (fileProperties.get(Constants.URI).trim().isEmpty()) {
+        } else if (fileProperties.get(FtpConstants.URI).trim().isEmpty()) {
             final RemoteFileSystemConnectorException e = new RemoteFileSystemConnectorException(
-                    "[" + serviceName + "] " + Constants.URI
+                    "[" + serviceName + "] " + FtpConstants.URI
```

Comment:
Shall we use `String.format` to create error messages rather than `String` concatenation ?

Reply:
For the moment shall we keep this implementation?